packages/typescript/plugin.js

```diff
@@ -0,0 +1,19 @@
+Plugin.registerCompiler({
+  extensions: ["ts", "tsx"],
+}, function () {
+  return new TypeScriptCompiler({
+    react: true,
+    typescript: true,
```

Comment:
See https://github.com/meteor/babel/pull/25 for details about how this is implemented in `meteor-babel`.